VSOWorkBot/Interfaces/IVsoApiController.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace VSOWorkBot.Interfaces
+{
+using Microsoft.TeamFoundation.WorkItemTracking.WebApi.Models;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using VSOWorkBot.Models;
+using WorkItemType = Models.WorkItemType;
+
+public interface IVsoApiController
```

Comment:
Why do you need an interface?

Reply:
When adding tests we can easily mock it to check dialog flow